satpy/etc/enhancements/atms.yaml

```diff
@@ -0,0 +1,29 @@
+enhancements:
+
+  mw183_humidity:
+    standard_name: mw183_humidity
+    operations:
+    - name: inverse
+      method: !!python/name:satpy.enhancements.invert
+      args:
+      - [true, true, true]
+    - name: stretch
+      method: !!python/name:satpy.enhancements.stretch
+      kwargs: {stretch: linear}
```

Comment:
Would it be possible to have a static stretch for this and the other composite? That way the inversion wouldn't be needed and the whole data (which might not be that large?) wouldn't need to be readed for determining the stretching.

Reply:
Good point, let me see...

Reply:
Okay, I agree t could make sense here to make the stretch static. But please be aware that I have not spend a great deal of research on how to combine the different channels in the best way to make it useful for a forecaster. But thought it was good to have something included here, then someone else (maybe a later me) can elaborate more and suggest adequate RGB types. I included one with the window channels and one with only the absorbing channels.
And I think I did as you suggested?

Reply:
Looking great for such a quick addition, thanks!

Are there any references for these composites, or did you just invent them on-the-fly?

Reply:
My invention so far. Haven't seen such RGBs before...
That is also why I am sure that more "research" is possible here!

Reply:
Well invented in that case!